### PR TITLE
Fix signedness warnings in LV2EffectProcessor for armv7hl

### PR DESCRIPTION
**Pull request by uklotzde**
http://koji.rpmfusion.org/kojifiles/work/tasks/4398/444398/build.log

### CODE REVIEW

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you. Waiting for CI.

**Comment by uklotzde** [comment]
Merging this trivial change now after approval.

**Pull request merged**
